src/middlewared/middlewared/plugins/sysdataset.py

```diff
@@ -416,6 +416,9 @@ async def migrate(self, _from, _to):
             path = '/tmp/system.new'
             if not os.path.exists('/tmp/system.new'):
                 os.mkdir('/tmp/system.new')
+            else:
+                # Make sure we clean up any previous attempts
+                await run('umount', '-R', path, check=False)
```

Comment:
Does FreeBSD have this option?

Reply:
Dunno, also not relevant anymore

Reply:
Does that mean we don't need FreeBSD code in `master` anymore?